python_modules/dagster/dagster/__init__.py

```diff
@@ -564,7 +564,9 @@
     # from dagster.some.module import (
     #     Foo as Foo,
     # )
-    pass
+    from dagster._core.execution.job_execution_result import (
+        JobExecutionResult as ExecuteJobResult,  # noqa: F401
+    )
```

Comment:
From summary:

>Alias JobExecutionResult to ExecuteJobResult at the import level for backcompat. Arguably we don't need this since execute_job is experimental, but I've included it.

Reply:
i meant add comment to the code

Reply:
🤦 added